kodi_addon_checker/schema_validation.py

```diff
@@ -0,0 +1,47 @@
+import os
+import xmlschema
+import logging
+
+from .report import Report
+from .record import PROBLEM, Record, INFORMATION
+
+xml_schema = "./xml_schema/"
+LOGGER = logging.getLogger(__name__)
+
+
+def schemas(report: Report, parsed_xml):
+    metadatacount = 0
+    validcount = 1
+    for extension in parsed_xml.findall("extension"):
+        extension_point = (extension.get("point")).split(".")[-1]
+        if extension_point == "metadata":
+            metadatacount += 1
+
+        schema_path = xml_schema + extension_point + '.xsd'
```

Comment:
I still don't like this use `os.path.join` to build the final path for the xsd file. Does `xml_schema = "./xml_schema/"` even work on windows? Obtain the relative path to the schema folder using the `os` module too